TombEngine/Objects/TR3/Trap/ElectricCleaner.cpp

```diff
@@ -0,0 +1,426 @@
+#include "framework.h"
+#include "Objects/TR3/Trap/ElectricCleaner.h"
+
+#include "Game/collision/collide_item.h"
+#include "Game/control/box.h"
+//#include "Game/effects/debris.h"
```

Comment:
If these includes are unneeded, they can be removed.

Reply:
Done